metaseq/launcher/slurm.py

```diff
@@ -461,11 +580,19 @@ def launch_train(args, grid, grid_product, dry_run, postprocess_hyperparams):
                 )
             else:
                 write_git_commit(train_log)
+                if args.rsc:
+                    klist(train_log)
                 with open(train_log, "a") as train_log_h:
                     job_id, stdout = run_batch(env, sbatch_cmd_str, sbatch_cmd)
                     print(stdout, file=train_log_h)
         if job_id is not None:
             print("Launched {}".format(job_id))
+        if args.sequential and not args.local and job_id is not None:
```

Comment:
what is this lol

Reply:
Do you know if the other things here are actually still used by people?
post_cmds
container_image and container_save - will remove
array_length
args.dep and args.sequential - will remove

Reply:
Containers might have been an external contributor except then why is it in metaseq-internal? Can we gut blame and try to figure out the why here? I'm afraid I may be missing context

Reply:
Containers might have been an external contributor except then why is it in metaseq-internal? Can we gut blame and try to figure out the why here? I'm afraid I may be missing context

Reply:
https://github.com/fairinternal/metaseq-internal/commit/2fb31531c85d0f82eaaea9d4d463e60afc4a74c5
Seems like this was for an old training run on azure. So I'll remove it as well.